test/hotspot/jtreg/runtime/Unsafe/Reallocate.java

```diff
@@ -59,7 +62,7 @@ public static void main(String args[]) throws Exception {
 
         // Make sure we can throw an OOME when we fail to reallocate due to OOM
         try {
-            unsafe.reallocateMemory(address, 100 * 1024 * 1024 * 8);
+            unsafe.reallocateMemory(address, 100 * 1024 * 1024);
```

Comment:
Why are we changing the test here? Just because 800m is a lot larger than the required 100m?

Reply:
MallocMaxTestWords took the value in *words*, that's where the 8 comes in. Note that it was broken anyway since it never shrank - it ignored free calls. Therefore the values given to MallocMaxTestWords have only ever been guesswork.